### DISCUSSION TITLE
Unhandled game packets: 118 and 11B

### DISCUSSION THREAD

**Discussion by Gradius2** [topaz-next/topaz#2428] [Help + Q&A] [answered]
Hi,

I saw this one since 2018 (perhaps they are even from 2017 or early).

And I always wanted to know, as they are still there (after all those years). 

What are those packets anyway?

[Warning] parse: Unhandled game packet 118 and 11B

**Comment by zach2good** [comment, marked as answer]
If you're ever interested in what certain packets are/do, Windower keeps good documentation on them.
https://github.com/Windower/Lua/blob/dev/addons/libs/packets/data.lua

Important to note: Windower is client-side, so outgoing are things the server receives and incoming are things the server sends. In this case:

https://github.com/Windower/Lua/blob/dev/addons/libs/packets/data.lua#L117
0x118 = 'Unity Chat Status', Sent when changing unity chat status.

They don't have an entry for 11B, so we're not entirely sure what that is 🤷 

EDIT:
We don't have unity chat, so there isn't any rush to implement handling of its related packets. If you don't like the log-spam, you can change your logging mask in `conf/map.conf`

**Comment by Gradius2** [comment]
Ahh I see.

Good to know.  :-)
